UnityProject/Assets/Scripts/Items/Tool/Soap.cs

```diff
@@ -33,6 +33,24 @@ public bool WillInteract(PositionalHandApply interaction, NetworkSide side)
 			//can only scrub tiles, for now
 			if (!Validations.HasComponent<InteractableTiles>(interaction.TargetObject)) return false;
 
+            // Get position of interaction as Vector3Int
+            Vector3 position = interaction.WorldPositionTarget;
+            Vector3Int positionInt = Vector3Int.RoundToInt(position);
+
+            // Check if there is an object in the way of scrubbing the tile
+			var atPosition = MatrixManager.GetAt<RegisterObject>(positionInt, true);
```

Comment:
```suggestion
			var atPosition = MatrixManager.GetAt<RegisterObject>(positionInt, side == NetworkSide.Server);
```
Sorry missed that one